src/js/widgets/SpacingPicker.js

```diff
@@ -0,0 +1,51 @@
+(function () {
+  var ns = $.namespace('pskl.widgets');
+
+  ns.SpacingPicker = function (onChange) {
```

Comment:
I think we could reuse the SizePicker widget directly. The issue is mostly with the hardcoded "4" used in setSpacing. The goal was to represent the maximum value that can be represented via a [data-size] item. 

Instead of doing that we can change the logic to do:
- get the element for the provided size `this.container.querySelector('[data-size="' + size + '"]')`
- if the element is missing, get the last element and add a label:
```
      selectedOption = this.container.querySelector('[data-size]:last-child');
      selectedOption.classList.add('labeled');
      selectedOption.setAttribute('real-size', size);
```

With this small change we can reuse the SizePicker directly, just need to change the setSpacing to setSize and the data-spacing to data-size where appropriate.